pallets/eth-bridge/src/lib.rs

```diff
@@ -195,6 +195,12 @@ pub mod pallet {
                 BoundedVec<(BoundedVec<u8, TypeLimit>, BoundedVec<u8, ValueLimit>), ParamsLimit>,
             caller_id: BoundedVec<u8, CallerIdLimit>,
         },
+        SetUpActiveRequest {
```

Comment:
why is this needed here?

Reply:
I added this as a way to verify which validator was responsible for the request. Thought it might be useful but can remove it if not needed.